### PR TITLE
Fixed ha_setup.yaml file

### PR DESCRIPTION
**Pull request by indrajitzagade**
## Problem Statement
<pre>
  <code>
  Story Ref (if any):
    Fix ha_setup.yaml file
  </code>
</pre>
## Unit testing on RPM done
<pre>
  <code>
  Yes
  </code>
</pre>
## Problem Description
<pre>
  <code>
     Fix ha_setup.yaml file
  </code>
</pre>
## Solution
<pre>
  <code>
    Fixed ha_setup.yaml file
  </code>
</pre>
## Unit Test Cases
<pre>
  <code>
    Ran -h and post_install command.
  </code>
</pre>

### CODE REVIEW

File: conf/mini_provisioner/v2/ha_setup.yaml [hunk 1]
```diff
@@ -16,38 +16,38 @@
...
         cmd: ha_setup backup
-        args: --location <URL>
+        args: --location $URL
 
     restore:
         cmd: ha_setup restore
-        args: --location <URL>
+        args: --location $URL
 
     support_bundle:
```
Comment (nitin-seagate) [inline conf/mini_provisioner/v2/ha_setup.yaml:53]: Support bundle is at same level as HA (I mean its not sub-key under HA)

**Review by nitin-seagate — APPROVED** [review]
Minor change with support bundle...Else looks good to me

File: conf/mini_provisioner/v2/ha_setup.yaml (hunk 1 above)
Comment (indrajitzagade) [reply conf/mini_provisioner/v2/ha_setup.yaml]: done

**Review by ajaysrivas — APPROVED** [review]

**Review by azheregelya — APPROVED** [review]

**Review by nitin-seagate — APPROVED** [review]
Looks good to me

File: conf/mini_provisioner/v2/ha_setup.yaml (hunk 1 above)
Comment (nitin-seagate) [reply conf/mini_provisioner/v2/ha_setup.yaml]: Thanks... Approved

**Pull request merged**